lib/rules/unit-case/index.js

```diff
@@ -60,9 +60,13 @@ const rule = function(expectation) {
       });
     }
 
-    root.walkAtRules(/^media$/i, atRule =>
-      check(atRule, atRule.params, atRuleParamIndex)
-    );
+    root.walkAtRules(atRule => {
+      if (!/^media$/i.test(atRule.name) && !atRule.variable) {
```

Comment:
There were a bunch of tests for `@variable` syntax, hence this change even though it's non-standard syntax. Perhaps those tests should be updated instead?